### ISSUE TITLE
discrete dimension differs continous one

### ISSUE THREAD

**Issue by denis-yeremich** [abeylot/geodataserver#5]
size_x == size_y == 256 in current config, but viewBox must be 0 0 255 255 because in discrete case size_x = x_max - x_min + 1 (not x_max - x_min)

https://github.com/abeylot/geodataserver/blame/4ed4d2781690b29e9f57f4683abb9eaffb927121/services/renderers/SvgRenderer.cpp#L363C5-L363C5

**Comment by denis-yeremich** [comment]
Also here
https://github.com/abeylot/geodataserver/blob/4ed4d2781690b29e9f57f4683abb9eaffb927121/services/renderers/SvgRenderer.cpp#L584
Thinking that rect 0 0 255 255 must be there

**Comment by abeylot** [comment]
I disagree, SVG logic isn't discrete, but maybe I'm wrong. I'll let the bug open for a while to have time to think about it.
